### DISCUSSION TITLE
Moins utiliser OnInit et OnChanges | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#22] [General]
# Moins utiliser OnInit et OnChanges | Bonjour Angular

Moins utiliser OnInit et OnChanges

https://apprendre.bonjour-angular.com/tips/moins-utiliser-oninit-onchanges/

**Comment by Neime** [comment]
Hello, comment utiliser ce fonctionnement avec getUserDetails qui returne une Promise ?
set userId, ne peut pas être async
